### DISCUSSION TITLE
Always dark color scheme

### DISCUSSION THREAD

**Discussion by joggienl** [sbarex/SourceCodeSyntaxHighlight#213] [Q&A] [answered]
Is there a way to force the color scheme to either light or dark, despite the OS settings?

**Comment by sbarex** [comment, marked as answer]
Yes, it is possible. From the theme selector popover, in the lower left corner is the icon to access the search criteria to choose only light or dark themes.

**Reply by joggienl** [reply]
hi @sbarex thanks!

That does work indeed! Not the most intuitive but it works fine!

Thanks!
